src/main/java/com/palantir/docker/compose/logging/LogDirectory.java

```diff
@@ -25,6 +25,11 @@ public static String circleAwareLogDirectory(Class<?> testClass) {
         return artifactRoot + "/dockerLogs/" + testClass.getSimpleName();
```

Comment:
Now that you've added the string one, what about passing using it here...

```java
return circleAwareLogDirectory(testClass.getSimpleName());
```

less code duplication!